pyfolio/plotting.py

```diff
@@ -510,13 +511,13 @@ def show_perf_stats(returns, factor_returns, live_start_date=None):
     perf_stats = np.round(timeseries.perf_stats(
         returns_backtest,
         factor_returns=factor_returns), 2)
-    perf_stats.columns = ['Backtest']
 
     if live_start_date is not None:
-        perf_stats = perf_stats.join(perf_stats_live,
-                                     how='inner')
-        perf_stats = perf_stats.join(perf_stats_all,
-                                     how='inner')
+        perf_stats = pd.DataFrame(OrderedDict([
```

Comment:
This is cool, I didn't know dataframe respected ordered dict key ordering until now